nilearn/plotting/_utils.py

```diff
@@ -55,8 +55,28 @@ def save_figure_if_needed(fig, output_file):
     return None
 
 
-def get_cbar_ticks(vmin, vmax, offset, n_ticks=5):
-    """Help for BaseSlicer."""
+def get_cbar_ticks(vmin, vmax, threshold=None, n_ticks=5):
+    """Return an array of ``n_ticks`` tick values to be used for the
+    colorbar.
+
+    Parameters
+    ----------
+    vmin: :obj:`float`
+        minimum value for the colorbar
+    vmax: :obj:`float`
+        maximum value for the colorbar
+    threshold: :obj:`float` or :obj:`int`
```

Comment:
```suggestion
    threshold: :obj:`float`, :obj:`int` or None
```

probably worth mentioning what happens when it's None

Reply:
Updated the docstring for `get_cbar_ticks` as:

```Return an array of evenly spaced ``n_ticks`` tick values to be used for
    the colorbar.
```

and for threshold:

```
if threshold is not None, ``-threshold`` and ``threshold`` values are replaced with the closest tick values
```

Would that be ok?